third_party/nvidia/hopper/lib/Transforms/WarpSpecialization/WSSpecialize.cpp

```diff
@@ -37,7 +37,14 @@ namespace mlir {
 
 Operation *SpecializeOp(Operation *op, IRMapping &mapping,
                         OpBuilderWithAsyncTaskIds &builder,
-                        AsyncTaskId asyncTaskId);
+                        SmallVector<AsyncTaskId> taskList);
+
+static bool hasAsyncTaskId(Operation *op, SmallVector<AsyncTaskId> &taskList) {
```

Comment:
Can this be replaced with an exisitng API in utility.cpp? If not, add it as a new API there?